pipelines/tasks/upload_prod_database.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+from pathlib import Path
+
+from pipelines.utils.storage_client import ObjectStorageClient
+from pipelines.utils.utils import get_project_root
+
+logger = logging.getLogger(__name__)
+
+
+def upload():
```

Comment:
```suggestion
def upload_duckdb_to_S3():
```
ou quelque chose comme cela